### PR TITLE
added datasets page

### PR DESCRIPTION
**Pull request by vincentberenz**
[//]: # "Thanks for your contribution.  To make life of the reviewers easier,"
[//]: # "please give this pull request a meaningful title and provide the"
[//]: # "requested information below."

## Description

Added the dataset page including info about the "Data of long-term experiment using a robot actuated by pneumatic muscles"

## How I Tested

built locally

### CODE REVIEW

File: D1_datasets.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The data encompasses movements generated by random multisine signals of target pressures, as well as repeated fixed target pressure movements, facilitating the evaluation of system repeatability and performance.
+
+### Robot
+
+The dataset was taken on Pamy2
+
+### Files
+
+The files can be downloaded on [Edmond](https://edmond.mpdl.mpg.de/dataset.xhtml?persistentId=doi:10.17617/3.OMM0JP).
+There are 6 zip files. Each files contains several "df" files (around 40), each file containning a pandas dataframe.
```
Comment (simonguist) [inline D1_datasets.md]: Small spelling issues:
- Each file contains (not files)
- each file containing (not containning)

File: D1_datasets.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- move to random pressure: Motion generate by applying a random target pressure.
+- reset: Reset motion by moving to medium pressures, then to min pressures, and back to medium pressures again; the final position might be suitable for assessing the system's repeatability.
+
+```python
+# see the categories
+dataframe.attrs.keys()
+
+# getting the intervals corresponding to "fixed movement fast"
+# 'fast_intervals' is a list of tuple (iteration start, iteration end)
+fast_intervals = df.attrs['fixed_movement_fast']
```
Comment (simonguist) [inline D1_datasets.md]: I think there is an inconsistency between dataframe and df

**Review by simonguist — APPROVED** [review]
Looks good

**Comment by vincentberenz** [comment]
I fixed the typos !

**Pull request merged**